aips/aip-31.md

```diff
@@ -37,6 +45,10 @@ Some delegation pools may require a KYC process before accepting delegated stake
 
 Delegation pools with allowlisting will have to manage their own KYC process. 
 
+The minimum stake requirement for active and pending-inactive stake per delegator is 10 APT.
+A delegator owning exactly 10 APT active stake might not be able to unlock or be evicted by the pool owner: 10 APT could be worth 9,99..9 APT when unlocked due to rounding errors caused by the "active to pending-inactive" conversion and the unlock would revert.\
+As rewards are added on this delegator's active stake, the active balance becomes > 10 APT which unblocks them. However, in case the validator is inactive and the delegator is not allowlisted, they don't produce rewards and also cannot add stake to get unblocked.
```

Comment:
how will a delegator know if they've been evicted from the allowlist?

Reply:
Individual evicts are not explicitly stored on-chain, they are only logged as events. A delegator in a specific pool finds out that they've been evicted by checking that they are not part of the allowlist and their entire active stake have been unlocked, but not by themselves.